src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -233,8 +260,8 @@ class CommandRequestImpl : public CommandImpl,
    * @return true if result code complies successful result code
    * otherwise returns false
    */
-  bool PrepareResultForMobileResponse(ResponseInfo& out_first,
-                                      ResponseInfo& out_second) const;
+  virtual bool PrepareResultForMobileResponse(ResponseInfo& out_first,
```

Comment:
@AByzhynar done in https://github.com/smartdevicelink/sdl_core/pull/1575/commits/596d8b9662b5da0b4c97f5d377d27bdfc0dfe16a